.github/workflows/build-engine-container.yml

```
@@ -11,9 +11,12 @@ on:
       version:
         description: "バージョン情報（A.BB.C / A.BB.C-preview.D）"
         required: true
+      skip_dockerhub:
+        description: "Docker Hubへのプッシュをスキップ"
```

変更箇所がSuggestionの外にもあったのでサジェスト適用ではなくコミットにしたんですが、descriptionも提案されていたことに気づきました。

その内容に再変更します。

```suggestion
        description: "Docker Hubへのプッシュやログインをスキップする"
```